<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainViewer</class>
 <widget class="QMainWindow" name="MainViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>739</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="glView" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>941</width>
      <height>531</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_move_x">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>560</y>
      <width>65</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-360.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_move_y">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>590</y>
      <width>65</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-360.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_move_z">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>620</y>
      <width>65</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-360.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_rotate_x">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>560</y>
      <width>65</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-360.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_rotate_y">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>590</y>
      <width>65</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-360.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_rotate_z">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>620</y>
      <width>65</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-360.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="init_x">
    <property name="geometry">
     <rect>
      <x>426</x>
      <y>560</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="init_y">
    <property name="geometry">
     <rect>
      <x>426</x>
      <y>590</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="init_z">
    <property name="geometry">
     <rect>
      <x>426</x>
      <y>620</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="movement">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>540</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MOVEMENT</string>
    </property>
   </widget>
   <widget class="QLabel" name="rotate">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>540</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> ROTATE</string>
    </property>
   </widget>
   <widget class="QLabel" name="file">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>           FILE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_file">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>             -</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_file">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>FILE .obj</string>
    </property>
   </widget>
   <widget class="QLabel" name="scaling">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>540</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>      SCALING</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SCREEN">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>590</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>SCREEN</string>
    </property>
   </widget>
   <widget class="QLabel" name="Vertices">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>VERTICES</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="count_vertices">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>490</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="count_edges">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>490</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Edges">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>490</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>EDGES</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="gif">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>560</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Gif</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="lineWidth">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>660</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="pointSize">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>660</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="hor_scale">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>560</y>
      <width>65</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="p_size">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>640</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>P_size</string>
    </property>
   </widget>
   <widget class="QLabel" name="line_w">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>640</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>line_widh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Color_Backg">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>540</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Color Background</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Color_Edge">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>570</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Color edge</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Color_Vertic">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>600</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Color vertices</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_no">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>540</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string> NO</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_Circle">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>570</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Circle</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_Square">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>600</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Square</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_Paralel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>540</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Paralel</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_Central">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>600</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Central</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_solid">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>590</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>solid</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_dashed">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>620</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>dashed</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>942</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>glView</class>
   <extends>QOpenGLWidget</extends>
   <header>headers/glview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
